src/main/resources/db/changelog/db.changelog-1.0.xml

```diff
@@ -0,0 +1,251 @@
+                                 constraintName="fk_authorisation_justification"
+                                 onDelete="NO ACTION" onUpdate="NO ACTION"/>
+        <addForeignKeyConstraint baseTableName="authorisation" baseColumnNames="meeting_date_id"
+                                 referencedTableName="meeting_date" referencedColumnNames="meeting_date_id"
+                                 constraintName="fk_authorisation_meeting_date"
+                                 onDelete="NO ACTION" onUpdate="NO ACTION"/>
+        <sql>
+            ALTER TABLE authorisation
+                ADD CONSTRAINT ck_authorisation_by_justification
+                    CHECK (justification_name = 'AUT' AND meeting_date_id IS NOT NULL AND aut_date IS NOT NULL AND kkt_check IS     NULL AND vir_date IS NULL     AND assurance_date IS NOT NULL
+                        OR justification_name = 'KKT' AND meeting_date_id IS NOT NULL AND aut_date IS NULL     AND kkt_check IS NOT NULL AND vir_date IS NULL     AND assurance_date IS NOT NULL
+                        OR justification_name = 'VIR' AND meeting_date_id IS NOT NULL AND aut_date IS NULL     AND kkt_check IS     NULL AND vir_date IS NOT NULL AND assurance_date IS NOT NULL
+                        OR justification_name = 'VIR' AND meeting_date_id IS NULL     AND aut_date IS NULL     AND kkt_check IS     NULL AND vir_date IS NOT NULL AND assurance_date IS NULL)
+        </sql>
+    </changeSet>
+
+    <changeSet id="2021-11-02-create-language_pair-table" author="terova">
+        <createTable tableName="language_pair">
+            <column autoIncrement="true" name="language_pair_id" type="BIGSERIAL">
+                <constraints primaryKey="true" primaryKeyName="language_pair_pkey"/>
+            </column>
+            <column name="version" type="INT" defaultValueNumeric="0">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="TEXT"/>
+            <column name="deleted_by" type="TEXT"/>
+            <column name="modified_by" type="TEXT"/>
+            <column name="creation_time" type="TIMESTAMP WITH TIME ZONE" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="modification_time" type="TIMESTAMP WITH TIME ZONE" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="deletion_time" type="TIMESTAMP WITH TIME ZONE"/>
+
+            <column name="authorisation_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="from_lang" type="VARCHAR(10)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="to_lang" type="VARCHAR(10)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="permission_to_publish" type="BOOL"/>
```

Comment:
Ehkä vaan `is_public` ? Toki julkisuus nyt riippuu siitä, löytyykö rekisteristä vast. kääntäjää ylipäätään.

Reply:
lupa julkaista vs on julkinen, mielestäni alkuperäinen kuvaa tarkemmin mistä tiedosta on kyse.

Reply:
Mennään vaan itse asiassa nykynimennällä juuri tästä syystä. Tälle sarakkeelle voisi tosin lisätä pakolliisuuden, että ei null. Null vastaisi tilaa, että ei tiedossa, saako julkaista vai ei. Toisaalta mietitkö asiaa tuota kautta muutoinkin?

Reply:
Tuotahan se null tarkoittaisi. Mutta taitaa olla tarpeetonta, lisään not null.